src/share/vm/jfr/leakprofiler/chains/edgeStore.cpp

```diff
@@ -75,22 +80,21 @@ bool EdgeStore::contains(const oop* reference) const {
 
 StoredEdge* EdgeStore::get(const oop* reference) const {
   assert(reference != NULL, "invariant");
-  const StoredEdge e(NULL, reference);
-  EdgeEntry* const entry = _edges->lookup_only(e, (uintptr_t)reference);
+  EdgeEntry* const entry = _edges->lookup_only((uintptr_t)reference);
   return entry != NULL ? entry->literal_addr() : NULL;
 }
 
 StoredEdge* EdgeStore::put(const oop* reference) {
```

Comment:
Does gc scan EdgeStore as root set?

Reply:
this operation is invoked  at safepoint.